src/validation_workflow/validation_args.py

```diff
@@ -85,6 +96,21 @@ def __init__(self) -> None:
             choices=["x64", "arm64"],
             default="x64"
         )
+        parser.add_argument(
+            "-p",
+            "--projects",
+            nargs='+',
+            help="Enter type of projects to be validated",
+            choices=["opensearch", "opensearch-dashboards"],
+            default=["opensearch"]
+        )
+        parser.add_argument(
+            "--artifact-type",
+            help="Enter type of artifacts that needs to be validated",
+            choices=["staging", "production"],
```

Comment:
What does the user need to enter for local artifact? It is neither staging, nor prod.

Reply:
Staging or prod is not required this comes under only providing the --file_path argument similar to file-path opensearch="src/opensearch.tar.gz" opensearch-dashboards="src/opensearch-dashboards.tar.gz". We distinguish the local artifacts based on not having `https` inside the provided string.